packages/datagrid/src/components/DefaultHeaderRenderer/QualityBar.component.js

```diff
@@ -38,7 +39,18 @@ function QualityBar(props) {
 						percentage: props.invalid.percentage,
 						total: formatNumber(props.invalid.total),
 					})}
-				/>
+				>
+					{ props.invalid.buttonMenu && <MenuDropDownQualityBar {...props.invalid.buttonMenu} />}
```

Comment:
Maybe we could consider using the QualityBar component coming from @talend/react-components as we are there

Reply:
i'm not sure, because it's a component's datagrid. For me this component will be localised in datagrid package, no ?